_posts/2018-7-18-EIA-coal.md

```diff
@@ -0,0 +1,141 @@
+
+![Coal Production]({{ site.baseurl }}public/images/CoalProduction.gif)
+<sup>_Yearly coal production for electricity usage, shown at both mine and state level._</sup>
+
+As someone who grew up in the US, this map challenges the common mythology I often heard about [Appalachia and coal mining](https://en.wikipedia.org/wiki/Appalachia#Coal_mining){:target="_blank"}, and assumptions about states such as West Virginia being dominant producers.  While Appalachia was a primary source of coal historically, it has been far eclipsed by Wyoming's [Powder River Basin](https://www.scientificamerican.com/article/powder-rive-basin-coal-on-the-move/){:target="_blank"}.
+
+The Powder River Basin is composed of surface mines with giant coal seams as shown below.  This region produces around 42% of all coal in the US[^PowderRiverBasin_42_percent], with the North Antelope Rochelle Mine alone providing 12% of total US production in 2016[^NorthAntelope_12_percent].  This single mine produces more coal than West Virginia, the second largest coal mining state.
+
+![Coal Seam]({{ site.baseurl }}public/images/Coal_Seam_at_Coal_Mine.png)
+<sup>_Coal seam, [North Antelope Rochelle Mine](https://en.wikipedia.org/wiki/North_Antelope_Rochelle_Mine){:target="_blank"}. By Peabody Energy, Inc. [CC BY 3.0](https://creativecommons.org/licenses/by/3.0){:target="_blank"}, via [Wikimedia Commons](https://commons.wikimedia.org/wiki/File:Coal_Seam_at_Coal_Mine.png){:target="_blank"}_</sup>
+
+The animated map below by the [Google Earth Engine Timelapse](https://earthengine.google.com/timelapse/#v=43.60679,-105.24646,9.613,latLng&t=2.96){:target="_blank"} shows the enormous geographic scale of the Power River Basin mines, along with their historical growth over 32 years of satellite imagery.  Over time, one can see new areas being dug out with land restoration efforts following shortly behind.
+
+{% raw %}
+<div>
+<iframe style="position: absolute; top: -9999em; visibility: hidden;" onload="this.style.position='static'; this.style.visibility='visible';" src="https://earthengine.google.com/iframes/timelapse_player_embed.html#v=43.60679,-105.24646,9.826,latLng&t=1.82" width="700" height="500"></iframe>
+</div>
+{% endraw %}
+<sup>_Coal Mines, Powder River Basin, Wyoming, [Google Earth Engine Timelapse](https://earthengine.google.com/timelapse/#v=43.60679,-105.24646,9.613,latLng&t=2.96){:target="_blank"}_</sup>
+
+## Sulfur Content
+
+The EIA data on coal shipments has incredible resolution, and one can find information about quarterly shipments between individual mines and power plants.  For each of these there is information about the type of coal, ash content, heat content, price, quantity, and sulfur content.
```

Comment:
--> "The EIA data on coal shipments has incredible resolution, including information about quarterly shipments between _individual_ mines and power plants."